### PR TITLE
Add new Goods and Artist Store merchants

### PR DESCRIPTION
**Pull request by krackydile**
A variety of new merchants now officially accept Monero as a form of payment. Please add them to your list.

### CODE REVIEW

File: _data/merchants.yml [hunk 1]
```diff
@@ -189,6 +189,8 @@
   merchants:
     - name: All For Crypto
       url: https://allforcrypto.com
+    - name: Cannabis Cup
+      url: https://shop.cannabiscup.com/
```
Comment (erciccione) [inline _data/merchants.yml]: I couldn't find written anywhere that they accept Monero, can you provide a link where they explicitly say they accept it?

File: _data/merchants.yml [hunk 2]
```diff
@@ -228,3 +234,96 @@
       url: https://safedice.com
     - name: XMRLotto.com
       url: https://xmrlotto.com
+- category: Artist Stores
```
Comment (erciccione) [inline _data/merchants.yml]: I don't like the idea of adding every single artist. This page is basically text only, adding a list of names will only make  it harder to read and ugly. Can we just put http://www.projectcoralreef.com/all-stores/ , who contains all the links to the artist's shops?

File: _data/merchants.yml [hunk 3]
```diff
@@ -209,6 +211,10 @@
       url: http://investmentart.org/
     - name: Mushroom cultures, mushroom growing supplies, seeds
       url: https://www.vesp.co/Fungible
+    - name: Plazmatic
+      url: http://www.plazmatic.com
+    - name: Project Coral Reef Store
+      url: https://shop.projectcoralreef.com
```
Comment (erciccione) [inline _data/merchants.yml]: Would be better to change this to `http://www.projectcoralreef.com/all-stores/`, see following comment.

File: _data/merchants.yml [hunk 4]
```diff
@@ -209,6 +211,10 @@
       url: http://investmentart.org/
     - name: Mushroom cultures, mushroom growing supplies, seeds
       url: https://www.vesp.co/Fungible
+    - name: Plazmatic
+      url: http://www.plazmatic.com
```
Comment (erciccione) [inline _data/merchants.yml]: Same of above, Monero is not in the accepted forms of payment: https://www.plazmatic.com/pages/faq

File: _data/merchants.yml (hunk 2 above)
Comment (krackydile) [reply _data/merchants.yml]: Agreed. I'll revert this clutter and add that as a new merchant in the merchant category

**Comment by luigi1111** [comment]
@krackydile Please squash.

**Comment by krackydile** [comment]
Awesome. Thanks all.

**Comment by krackydile** [comment]
@luigi1111 Sorry, was the squash correct? Haven't had to do that before.

**Comment by mattcode55** [comment]
@krackydile try `git rebase -i HEAD~10` and replacing all but the first `pick` with `squash`. https://git-scm.com/docs/git-rebase#_interactive_mode. You might have to `git push --force` afterwards.

**Comment by krackydile** [comment]
@mattcode55 Not sure what's happening now. But I did get my update to be the last line.

**Comment by mattcode55** [comment]
The idea is that all of your changes get put into one commit to keep things tidy.

It looks like everything got a bit crazy when you merged master back into your branch.

**Comment by luigi1111** [comment]
Now it's much, much worse. :)

**Comment by krackydile** [comment]
@luigi1111 Seriously is. Sorry for the trouble. I'm thinking it might be easier and cleaner to just close this PR and start from a clean slated one and add my two lines of updates.

**Comment by mattcode55** [comment]
~~Maybe this is why it's called Git.~~ 

> I'm thinking it might be easier and cleaner to just close this PR and start from a clean slated one and add my two lines of updates.

This PR is still salvageable, but you can save yourself a headache by making a new one.

**Comment by luigi1111** [comment]
@krackydile are you on IRC? There are plenty of good folks in #monero-dev that would be happy to help you.

**Comment by krackydile** [comment]
@luigi1111 I am not on IRC

**Pull request closed without merging**